examples/pubsub_scheduled_multiple/main.tf

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = ">= 0.12"
+}
+
+provider "google-beta" {
+  version = "~> 2.5"
+  project = var.project_id
+  region  = var.region
+}
+
+module "pubsub_scheduled_1" {
+  providers = {
+    google = google-beta
+  }
+
+  source                    = "../../"
+  project_id                = var.project_id
+  job_name                  = "pubsub-example"
+  job_schedule              = "*/5 * * * *"
+  function_entry_point      = "doSomething"
+  function_source_directory = "${path.module}/function_source"
+  function_name             = "testfunction-1"
+  region                    = var.region
+  topic_name                = "pubsub-1"
+}
+
+module "pubsub_scheduled_2" {
+  providers = {
+    google = google-beta
+  }
+
+  source                    = "../../"
+  project_id                = var.project_id
+  function_entry_point      = "doSomething"
+  function_source_directory = "${path.module}/function_source_2"
+  function_name             = "testfunction-2"
+  region                    = var.region
+  topic_name                = "pubsub-1"
```

Comment:
Since we can also share the same topic, this could be `topic_name = module.pubsub_scheduled_1.pubsub_topic_name`

Reply:
Agreed. Moreover, I think sharing the same topic is the only option if we have a single existing scheduler, because the [google_cloud_scheduler_job](https://www.terraform.io/docs/providers/google/r/cloud_scheduler_job.html) can have only one `pubsub_target` section. Described an alternative approach using multiple event-function module instances with a single scheduler and using the same pubsub topic name at https://github.com/terraform-google-modules/terraform-google-scheduled-function/issues/15#issuecomment-564426393. Following the approach we wouldn't need making any changes in the `terraform-google-scheduled-function` module.

Reply:
I get that it would force you to use the same topic, but this means we need to prevent the creation of the other topics (for module 2 and 3) somehow - hence `create_topic` set to `false` when an existing scheduler is passed to the module.

Reply:
We can prevent creating the `google_cloud_scheduler_job` (using `count = var.scheduler_job != "" ? 0 : 1`) and the `pubsub_topic` resource (using `create_topic = var.scheduler_job != "" ? false : true`) when an existing `scheduler_job` is passed, what do you think ?